src/Microsoft.Azure.SignalR.Management/ServiceManager.cs

```diff
@@ -29,11 +29,11 @@ internal class ServiceManager : IServiceManager
         private readonly string _productInfo;
         private readonly RestClientFactory _restClientFactory;
 
-        internal ServiceManager(ServiceManagerOptions serviceManagerOptions, string productInfo, RestClientFactory restClientFactory)
+        internal ServiceManager(ServiceManagerOptions serviceManagerOptions, ServiceManagerContext context, RestClientFactory restClientFactory)
```

Comment:
how about ServiceManager only relies on `ServiceManagerContext` and never `ServiceManagerOptions`?